.github/workflows/iroha2-deploy-base-image.yml

```
@@ -0,0 +1,81 @@
+name: I2::Base Image Deploy
+
+on:
+  push:
```

```suggestion
on:
  workflow_dispatch:
  push:
```
Just in case `archlinux:base-devel` is silently outdated